govuk_navigation_helpers.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "gds-api-adapters", "~> 42.0"
```

Comment:
Sounds vaguely dangerous, but the question is would it the the cool kind of dangerous or the scary kind of dangerous?